packages/vite-plugin-cloudflare/src/workers-configs.ts

```diff
@@ -351,8 +352,74 @@ export function getWorkerConfig(
 	};
 }
 
+/**
+ * Returns the path to a wrangler config for a worker after having it validated
+ * (throws appropriate errors in case the validation fails)
+ *
+ * @param root the root of the vite project
+ * @param requestedConfigPath the requested config path, if any
+ * @param isForAuxiliaryWorker whether the config path is being requested for an auxiliary worker
+ * @returns a valid path to a config file
+ */
+export function getValidatedWranglerConfigPath(
+	root: string,
+	requestedConfigPath?: string,
```

Comment:
This should always be provided.

```suggestion
	requestedConfigPath: string | undefined,
```